src/components/monitor/AllGoodSection.tsx

```diff
@@ -0,0 +1,19 @@
+import { HaveIbeenPwnedText } from '@/assets/types/have-i-been-pawned';
+import { CheckCircle, Smiley } from '@phosphor-icons/react/dist/ssr';
+
+export interface FeatureSectionProps {
+  textContent: HaveIbeenPwnedText['HeroSection']['AllGoodSection'];
+}
+export const AllGoodSection = ({ textContent }) => {
```

Comment:
But use the interface to type textContent. Also change the name to `AllGoodSectionProps`.

Reply:
Done